tools/policy-gen/crd-extract-openapi.sh

```diff
@@ -7,12 +7,13 @@ set -e
 
 POLICY=$1
 VERSION=${2:-"v1alpha1"}
+TOOLS_LOCATION=${3:-"."}
```

Comment:
I would rely on a default (make the param required).

You can then pass in your script: 

https://github.com/kumahq/kuma/blob/6f45254d7048e734317c723d433b94bcd406f02a/mk/dev.mk#L19

there:

https://github.com/kumahq/kuma/blob/6f45254d7048e734317c723d433b94bcd406f02a/mk/generate.mk#L61

I guess the same applies with VERSION as it's always set the default should be set.

Also should there be some more changes to the makefile to be able to reuse the targets in kong-mesh by just including instead of copying them? Seems like this would improve maintainability

Reply:
changed